### ISSUE TITLE
US Counties

### ISSUE THREAD

**Issue by tghkwomack** [CSSEGISandData/COVID-19#1111]
It would be nice to have the US Counties represented in a way that didn't include a comma.  It throws off attempts to load the data as a comma-delimited file using methods that don't readily allow a text qualifier

**Comment by jcmartel** [comment]
Where are you seeing data for US Counties? I was looking for it but just see select cities and counties in the master data files with the countries. Is there a separate file for US Counties?

**Comment by tghkwomack** [comment]
Thanks for your response!  I see it in the time series files (sample below).  The Confirmed, Recoveries and Deaths files.

[cid:image001.png@01D5FEA0.90F70D30]


Kevin Womack, MHA
Sr. Data Warehouse Analyst
Tampa General Hospital
813.844.5074 – Office
813.546.2156 – Cell
kwomack@tgh.org<mailto:kwomack@tgh.org>

We Heal. We Teach. We Innovate.
Care for everyone. Every day.
[imageTGH]<https://health.usnews.com/best-hospitals/area/fl/tampa-general-hospital-6391060>

**Comment by JiPiBi** [comment]
@tghkwomack  please dont resend the entire mail you received  , answer on the site

**Comment by GitHub4PeterRoss** [comment]
Suggestion - download csv, reorder columns, do "text to columns", pluck out non-US, and then pivot.  Working on a cleaner thought/approach and will share.

**Comment by dhmacq** [comment]
> It would be nice to have the US Counties represented in a way that didn't include a comma. It throws off attempts to load the data as a comma-delimited file using methods that don't readily allow a text qualifier

Actually, I just finished using the commas in code to extract the subset of counties. I'd rather the commas continue to be present.

I would support a change from comma delimiters to tab delimiters (csv to tsv). Excel and R both read the files without trouble. By "text qualifier" you mean putting quotes around cases where the text includes a comma?

**Comment by tghkwomack** [comment]
Yes, that's what I mean by text qualifier.  I'm using Bulk Insert to load the data into a table.  But, I'd rather not have to create a format file, and the version of SQL Server I'm using doesn't allow you to specify a text qualifier on the command line.  Regardless, a tab-delimited format would be a great alternative.  Thanks for your response!
